spec/lib/vacuum/invites_vacuum_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Vacuum::InvitesVacuum do
+  subject { described_class.new(retention_period, retention_max_uses) }
+
+  let(:retention_period) { 7.days }
+  let(:retention_max_uses) { 10 }
+
+  let(:user) { Fabricate(:user) }
+
+  describe '#perform' do
+    let!(:invite_unlimited) { Fabricate(:invite, user: user, max_uses: nil, created_at: 10.days.ago, expires_at: nil) }
+    let!(:invite_huge_max_uses) { Fabricate(:invite, max_uses: 100, created_at: 10.days.ago, expires_at: nil) }
+    let!(:invite_small_max_uses) { Fabricate(:invite, max_uses: 2, created_at: 10.days.ago, expires_at: nil) }
+    let!(:invite_will_expires_later) { Fabricate(:invite, max_uses: nil, created_at: 1.hour.ago, expires_at: 1.hour.from_now) }
+
+    before do
+      subject.perform
```

Comment:
I think as currently organized here, we'll wind up creating 20 factory records (5 lets across 4 examples).

Maybe adjust to have one example that starts with `expect { subject.perform }...` and then chain on all the expected updates with `change` blocks?